src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -195,6 +197,60 @@ public TutorialClass findTutorialClassFromList(TutorialClass tutorialClass, Modu
         return tutorialClassInList;
     }
 
+    /**
+     * Randomly allocates the students in {@code tutorial class} into {@code numOfTeams} of different teams.
+     *
+     * @param moduleCode of the tutorial class.
+     * @param tutorialClass to allocate the different students into the teams to.
+     * @param numOfTeams of teams to split into.
+     */
+    public void randomTeamAllocation(ModuleCode moduleCode, TutorialClass tutorialClass, int numOfTeams) {
+        requireAllNonNull(moduleCode, tutorialClass, numOfTeams);
+        ArrayList<TutorialTeam> teams = tutorialClass.getTeams();
+        ArrayList<Person> classList = tutorialClass.getStudents();
+        int classSize = tutorialClass.getStudents().size();
```

Comment:
Technically you can use classList.size() since you have already called the getStudents function so that you dont need to getStudents again

Reply:
Have made the necessary changes, thanks!